src/Desalt.TypeScriptAst/Ast/Grammar/Generated/TsAstNodes.Others.cs

```diff
@@ -1202,6 +1202,30 @@ public static class IdentifierExtensions
             node.Text == value ? node : new TsIdentifier(value, node.LeadingTrivia, node.TrailingTrivia);
     }
 
+    public interface IMemberAccessOmittedIdentifier : ITsAstNode, ITsExpression
```

Comment:
I know this is generated code—I am just trying to figure out what I wanted to generate.

Reply:
I'd appreciate your suggestions here.  I don't know if you want this node to be radically different from the others.